scaladoc/src/dotty/tools/scaladoc/tasty/SymOps.scala

```diff
@@ -119,21 +152,27 @@ class SymOps[Q <: Quotes](val q: Q):
           else if (sym.maybeOwner.isDefDef) Some(sym.owner)
           else None
 
-        val originPath = {
+        val className = sym.className
+
+        val location = className.fold(sym.packageName)(cn => s"${sym.packageName}.${cn}")
```

Comment:
I think we can work with Seq[Sring] rather then concatenate and then split it later on in each branch